Tools/autotest/default_params/rover-omni3mecanum.parm

```diff
@@ -0,0 +1,36 @@
+ACRO_TURN_RATE 300.000000
+ATC_SPEED_P 0.200000
+ATC_SPEED_D 0.000010
+ATC_STR_RAT_D 0.000010
+ATC_STR_RAT_FF 0.100000
+ATC_STR_RAT_I 0.010000
+ATC_STR_RAT_MAX 360.000000
+ATC_STR_RAT_P 0.030000
+CRUISE_SPEED 1.0
+CRUISE_THROTTLE 42.0
+FRAME_TYPE 4
+PSC_VEL_P 0.500000
+RC1_MAX 2000
```

Comment:
Hi @srmainwaring, thanks for this.  Normally we don't add items to the param file that are the same as the defaults.  I suspect we don't need these RCx_MIN/MAX values nor the SERVOx_MIN/TRIM/MAX

Reply:
Thanks @rmackay9, updated in https://github.com/ArduPilot/ardupilot/commit/0044bb56d5b5bf2d7e206e845de4d3d00eafedd9. All remaining params are different from defaults or not set in `rover.param`.